content/docs/gto/command-reference/annotate.md

```diff
@@ -0,0 +1,86 @@
+# annotate
+
+Update artifact metadata annotations.
+
+## Synopsis
+
+```usage
+usage: gto annotate [-r <text>] [--type <text>] [--path <text>]
+                    [-e] [--label <text>] [-d <text>] [--commit]
+                    [--push] [-h]
+                    name
+
+arguments:
+  name             Artifact name
+```
+
+## Description
+
+The command adds the artifact annotation to `artifacts.yaml` file. Feel free to
+modify the file directly!
+
+For example,
+
+```cli
+$ gto annotate awesome-model \
+  --type model \
+  --path models/neural_network.h5 \
+  --label ml
+  --label cool
+  --description "This model is very cool"
+```
+
+will create
+
+```yaml
+awesome-model:
+  description: This model is very cool
+  labels:
+    - ml
+    - cool
+  path: models/neural_network.h5
+  type: model
+```
+
+This information can be later retrieved by running `gto describe` command.
+
+> Don't forget to commit `artifacts.yaml` with Git to associate it with the
+> latest artifact version and stage in any copy of the repo.
+
+By default GTO saves artifact as `virtual`. Use the `--must_exist` flag to tell
+GTO the artifact file is committed to Git.
+
+<details>
+
+### Virtual vs. Physical artifacts
+
+- Physical files/directories are committed to the repo. When you create a new
+  version or assign a stage to it, Git guarantees that it's immutable -- you can
```

Comment:
- Minor or for a later follow-up: Ideally this deeper explanation would be in another section of the docs (in a guide, for example) and just briefly mentioned/explained here (linked to the guide).

Reply:
Should we move it to https://mlem.ai/doc/gto/user-guide#annotations-in-artifactsyaml ? Feel free to address any pending items here in #235.